### PR TITLE
amazon.aws/gate: temporarily disabled the integration job

### PR DESCRIPTION
**Pull request by goneri**
We've got a bunch of PR that we trust to land quickly. We will reverse
the PR once they are merged.

### CODE REVIEW

**Review by jillr — APPROVED** [review]
For gate this makes sense - thanks!

**Comment by goneri** [comment]
I suggest you set some Depends-On: on the PR that need to be merged. And once we've got enough PR ready we land this one first and gate the rest of the convoy.

**Comment by jillr** [comment]
@goneri  The only PR I have remaining is the release prep, which is waiting on reviews https://github.com/ansible-collections/amazon.aws/pull/489

**Pull request closed without merging**
